airflow/providers/google/cloud/operators/automl.py

```diff
@@ -246,7 +306,7 @@ def execute(self, context: Context):
             metadata=self.metadata,
         )
         project_id = self.project_id or hook.project_id
-        if project_id:
+        if project_id and self.model_id:
```

Comment:
Why is `self.model_id` now required?

Reply:
Since we introduce the `endpoint_id`, `model_id` is optional right now. This check is to ensure the `model_id` is given. Because `AutoMLModelPredictLink` can be used only for `model_id` case